src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java

```diff
@@ -356,6 +359,15 @@ private boolean isGTMachine(Object o) {
         return false;
     }
 
+    private boolean isFluidConduitConnected(TileEntity te, ForgeDirection dir) {
+        try {
+            ILiquidConduit liquidConduit = (ILiquidConduit) eioTypeCheck.invoke(te, ILiquidConduit.class);
```

Comment:
Is this really how this is normally used? Seems really strange to be catching multiple exceptions like that. I will look at this more.

Reply:
This seems fine. Just strange.